### ISSUE TITLE
Android resource linking failed for magicretry_fragment.xml

### ISSUE THREAD

**Issue by mvij** [payu-intrepos/magicretry#3]
When i try to build this lib using Android Studio 3.5.0, i get the following error
`AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/vijith/.gradle/caches/transforms-2/files-2.1/0c1b042f08a50764c3d60743091442f0/jetified-magicretry-1.0.7/res/layout-v11/magicretry_fragment.xml","position":{"startLine":120}}],"original":"/home/vijith/.gradle/caches/transforms-2/files-2.1/0c1b042f08a50764c3d60743091442f0/jetified-magicretry-1.0.7/res/layout-v11/magicretry_fragment.xml:121: AAPT: error: resource android:attr/android:progressBarStyleSmall not found.\n    ","tool":"AAPT"}`

**Comment by here4learning** [comment]
Any solution??

**Comment by mvij** [comment]
The layout file `magicretry_fragment.xml` in layout-v11 res folder has `?android:attr/android:progressBarStyleSmall` at line 126. It should be `?android:attr/progressBarStyleSmall` or `?attr:progressBarStyleSmall`

**Comment by here4learning** [comment]
yes.Correct. But it is not accessible to change the xml. . So it has to be done at Payu side.

**Comment by Ranjithsp6** [comment]
> yes.Correct. But it is not accessible to change the xml. . So it has to be done at Payu side.

have you found the temporary solution?

**Comment by here4learning** [comment]
Create a folder within main res folder and create the xml file according to last comment of the link " 
https://stackoverflow.com/questions/57588753/aapt-error-resource-androidattr-androidprogressbarstylesmall-not-found" . it will override payumoney xml file.

**Comment by Ranjithsp6** [comment]
> Create a folder within main res folder and create the xml file according to last comment of the link "
> https://stackoverflow.com/questions/57588753/aapt-error-resource-androidattr-androidprogressbarstylesmall-not-found" . it will override payumoney xml file.

Thank you its working

**Comment by himgupta229** [comment]
> When i try to build this lib using Android Studio 3.5.0, i get the following error
> `AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/vijith/.gradle/caches/transforms-2/files-2.1/0c1b042f08a50764c3d60743091442f0/jetified-magicretry-1.0.7/res/layout-v11/magicretry_fragment.xml","position":{"startLine":120}}],"original":"/home/vijith/.gradle/caches/transforms-2/files-2.1/0c1b042f08a50764c3d60743091442f0/jetified-magicretry-1.0.7/res/layout-v11/magicretry_fragment.xml:121: AAPT: error: resource android:attr/android:progressBarStyleSmall not found.\n ","tool":"AAPT"}`

What is the gradle plugin version you are using?

**Comment by jakal23** [comment]
Have the same problem migrating androidx
